channels.go

```diff
@@ -71,6 +76,7 @@ func (b *Bot) GetChatAdministrators(recipient Recipient) ([]ChannelUser, error)
 	return responseRecieved.Result, nil
 }
 
+// GetChatMembersCount... are you kidding me ? this method return number of users in a channel or group and etc...
```

Comment:
I think `// GetChatMembersCount get the number of members in a chat` will be more suitable for documentation.